core/src/migrations/000086-addDestinationRecordCollection.ts

```diff
@@ -0,0 +1,26 @@
+import Sequelize from "sequelize";
```

Comment:
Assuming https://github.com/grouparoo/grouparoo/pull/2341 comes in with migrations 84 and 85, this would be migration 86